cmd/endorse.go

```diff
@@ -126,6 +134,16 @@ func (f *endorseCommand) PersistentPreRunE(cmd *cobra.Command, _ []string) error
 	if !strings.HasSuffix(f.UefiPath, ".fd") {
 		return fmt.Errorf("--uefi=%q path must end with .fd", f.UefiPath)
 	}
+	if f.SvsmPath != "" {
+		if !strings.HasSuffix(f.SvsmPath, ".igvm") {
+			return fmt.Errorf("if specified, --svsm_path=%q path must end with .igvm", f.SvsmPath)
+		}
```

Comment:
Why is the extension relevant when the type is already known from using the flag?

Reply:
Deleted extension checks